src/main/java/com/google/impactdashboard/server/ProjectInformationRetriever.java

```diff
@@ -0,0 +1,64 @@
+package com.google.impactdashboard.server;
+
+import com.google.impactdashboard.data.project.Project;
+import com.google.impactdashboard.data.project.ProjectIdentification;
+import com.google.impactdashboard.data.project.ProjectMetaData;
+import com.google.impactdashboard.database_manager.data_read.DataReadManager;
+import com.google.impactdashboard.database_manager.data_read.DataReadManagerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Retrieves all the information about the projects in the database */
+public class ProjectInformationRetriever {
```

Comment:
It's not clear from the name and description what's the difference between ProjectDataRetriever and ProjectInformationRetriever.
Actually the functionality are very similar too. Could you maybe combine them into one class and different servlets can just use different methods of it?

Reply:
I guess I just wanted to keep everything separate but I guess they are so similar they can be in the same class.